pkg/flow/logging/sink.go

```diff
@@ -81,6 +85,10 @@ func (s *Sink) Update(o SinkOptions) error {
 func writerSinkLogger(w io.Writer, o SinkOptions) (log.Logger, error) {
 	var l log.Logger
 
+	if len(o.Fanout) > 0 {
+		w = &fanoutWriter{w: w, f: o.Fanout}
```

Comment:
Mutating the passed in `w io.Writer` feels off. Is there a cleaner way?

Reply:
I'll take another look if you'd like; I tried wrapping around the logger instead but this meant we didn't have access to the log level and log format, while this makes sure the logs in the pipeline are same ones you see in the console.

Reply:
Sounds reasonable, maybe add a comment to the funcs?